tests/5.1/scope/test_scope_nowait_construct.c

```diff
@@ -0,0 +1,49 @@
+//-------------test_scope_nowait_construct.c-------------//
+//
+//OpenMP API Version 5.1 Aug 2021
+//
+//Tests the behavior of the scope construct with nowait
+//specified. This test should remove the implied barrier
+//creatd by the scope construct when nowait is specified.
+//-------------------------------------------------------//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+#include <unistd.h>
+
+#define N 1024
+
+int test_scope_nowait(){
+	int errors = 0;
+	int test_arr[N];
+	for (int i=0; i<N, i++){
+		test_arr[i] = 1;
+		sum += i;
+	}
+	#pragma omp parallel shared(test_int)
+	{
+		#pragma omp scope nowait
+		{
+			for (int i=0; i<N; i++){
+				test_arr[i] += 1;
+			}
+		}
+	}
+	int sum = 0;
+	for (int i = 0; i < N; i++){
+		sum += test_arr[i];
+	}
+	OMPVV_TEST_AND_SET_VERBOSE(errors, sum != 2048);
```

Comment:
The expected value is not quite right. In the first loop, it is set to `sum += i` – which is not `N` (1024) but 523776 – should this be `sum += 1` instead?

However, even with the latter, it would not 2*N but 3*N as `test_arr[i] = 1` is followed by `test_arr[i] += 1` – such that `sum += test_arr[i]` adds 2*N – yielding `2*N + <value of sum of the first loop>` (which is either `2*N + 523776` or, after the 'sum += 1' change, `3*N`).